drf_spectacular/openapi.py

```diff
@@ -787,6 +787,32 @@ def _get_serializer(self):
                 f'a request? Ignoring the view for now. (Exception: {exc})'
             )
 
+    def get_examples(self):
+        return []
+
+    def _get_examples(self, serializer, direction, media_type, status_code=None):
```

Comment:
wow, It has become much much simpler than before 👍 

I Learned good things  thanks to your code review